docs/expressions.md

```diff
@@ -22,14 +22,15 @@ DataYoga supports both SQL and [JMESPath](https://jmespath.org/) expressions. JM
 | `hash`               | Calculates a hash using the `hash_name` hash function and returns its hexadecimal representation                                    | Input: `{"some_str": "some_value"}` <br /> Expression: `hash(some_str, &#96;sha1&#96;)` <br /> Output: `8c818171573b03feeae08b0b4ffeb6999e3afc05` | Supported algorithms: sha1 (default), sha256, md5, sha384, sha3_384, blake2b, sha512, sha3_224, sha224, sha3_256, sha3_512, blake2s                              |
 | `in`                 | Checks if an element matches any value in a list of values                                                                          | Input: `{"el": "b"}` <br /> Expression: `in(el, `["a", "b", "c"]`)` <br /> Output: `True`                                                         |                                                                                                                                                                  |
 | `left`               | Returns a specified number of characters from the start of a given text string                                                      | Input: `{"greeting": "hello world!"}` <br /> Expression: `left(greeting, `5`)` <br /> Output: `hello`                                             |                                                                                                                                                                  |
 | `lower`              | Converts all uppercase characters in a string into lowercase characters                                                             | Input: `{"fname": "John"}` <br /> Expression: `lower(fname)` <br /> Output: `john`                                                                |                                                                                                                                                                  |
 | `mid`                | Returns a specified number of characters from the middle of a given text string                                                     | Input: `{"greeting": "hello world!"}` <br /> Expression: `mid(greeting, `4`, `3`)` <br /> Output: `o w`                                           |                                                                                                                                                                  |
+| `parse_json`         | Generates a random UUID4 and returns it as a string in standard format                                                              | Input: `{"data": '{"greeting": "hello world!"}'}` <br /> Expression: `parse_json(data)` <br /> Output: `{"greeting": "hello world!"}`             |                                                                                                                                                                  |
```

Comment:
wrong description of this custom function

Reply:
✅